README.md

```diff
@@ -264,6 +272,10 @@ Renders the widget to the DOM, and passes control back to your app through succe
 - `success` *(optional)* - Function that is called when the user has completed an authentication flow. If an [OpenID Connect redirect flow](#openid-connect) is used, this function can be omitted.
 - `error` *(optional)* - Function that is called when the widget has been initialized with invalid config options, or has entered a state it cannot recover from. If omitted, a default function is used to output errors to the console.
 
+**OIDC Configuration:**
+
+This configuration is used for OIDC integrations. New integrations are recommended to use OIDC as it uses a more lightweight REST-based protocol and has more widespread usage.
```

Comment:
I pulled back the link I had here. It was related to OIDC/OAuth vs SAML protocol. I couldn't find anything better on the docs site that would help a user pick OIDC other than most docs use examples of OIDC integrations.